_articles/what-is-oauth-understanding-the-authorization-layer.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: article
+title: "What is OAuth? Understanding the authorization layer"
+abstract: "A simple explanation of what OAuth is, how it works and what it is used for."
+sitemap:
+  lastmod: 2020-05-15
+---
+
+OAuth is short for Open Authorization.  OAuth is a framework for token-based authorization on the internet. What does that mean?  Basically, OAuth is a way for websites to share private information with each other. Authorization tokens are issued which grants access to specific pieces of information hence the term token-based authorization.
+
+For example, let’s say you want your social media profile to share with a dating website your name, date of birth and height.  OAuth can be used to authorize the dating app access to those details on your social media profile.  The OAuth framework allows the social media profile to create an access token with certain attributes, such as an expiry date, which the dating website can use to access the data the token has scope for.
+
+This method allows websites to share only the specific pieces of information it chooses with other websites.  Before OAuth was invented, if two websites wanted to share information they had little choice but to share everything at an account level:
```

Comment:
Not it the website, you the user of that website

Reply:
> Before OAuth was invented, if two websites wanted to share information they had little choice but to share everything at an account level:

Where did you source this, it's not really what the problem was.

Reply:
The problem was you had to give you log in details for the social media site to the dating app, which was a problem if you didn't trust the dating app